Contest_API.md

```diff
@@ -1352,16 +1458,16 @@ endpoint returns with a `GET` request with the following exceptions:
 * Since `files` only supports `application/zip`, providing the `mime` property is
   optional.
 * `reaction` may be provided but a CCS does not have to honour it.
-* If the CCS supports a `team` role they will only have access to `POST`, and `time`
-  must not be provided when using this role. `time` will always be
-  set to the current time as determined by the server. `team_id` may be provided but then
+* The `team_submit` capability only has access to `POST`. `time`
+  must not be provided and will always be set to the
+  current time as determined by the server. `team_id` may be provided but then
   must match the ID of the team associated with the request.
-* Simple proxies using an `admin` role should also use `POST` to let the server
+* The `proxy_submit` capability also uses `POST` and lets the server
   determine the `id`. `team_id` must be provided but `time` must not and will be
   determined by the server.
-* For more advanced scenarios an `admin` role may use a `PUT`. `time` is required and
-  the client is responsible for making sure the provided `id` is unique. For example
-  in a setup with a central CCS with satellite sites
+* For more advanced scenarios the `admin_submit` capability may use a `POST` (must not
+  include an `id`) or `PUT` (client is required to include a unique `id`). In both
```

Comment:
So if you have `proxy_submit` you may provide `id`in a POST (which feels wrong in itself. If you are deciding the `id` you should use a PUT not a POST), but `admin_submit` may not. This feels inconsistent.

I suggest that neither can include an `id` in their POST, but both are allowed to PUT with an `id`.